index.html

```diff
@@ -723,10 +722,9 @@ <h2><code>application/vc+ld+jwt</code></h2>
       <h2><code>application/vp+ld+jwt</code></h2>
       <p>
         This specification registers the
-        <code>application/vp+ld+jwt</code> Media Type specifically for
+        <code>application/vp+ld+jwt</code> Media Type for
         identifying a <a data-cite="rfc7519#section-3">JWT</a>
-        conforming to the Verifiable Credentials JWT format in the `typ`
-        header.
+        Verifiable Presentation.
```

Comment:
In v1, a Verifiable Presentations is a JSON-LD document with a context and type, and a few other optional members.

Afaik, V2 has not changed any of this, so it's correct to call a JWT a Verifiable Presentation when the claimset is vp+ld+json.

Reply:
@OR13,

> Afaik, V2 has not changed any of this, so it's correct to call a JWT a Verifiable Presentation when the claimset is vp+ld+json.

The JWT may contain (or secure) a Verifiable Presentation, but it isn't one itself. A JWT is not "a JSON-LD document with a context and type and a few other optional members", but its payload could be. If someone were handed a JWT, they could not process it as a Verifiable Presentation. They would need to parse a `.`-delimited string of base64-encoded values (the JWT), then strip off the header component, the signature component, and just decode and parse the payload / claimset. It's just this latter bit that could be a Verifiable Presentation.

Reply:
https://w3c.github.io/vc-data-model/#securing-verifiable-credentials

> This specification recognizes two classes of securing mechanisms: those that use external proofs and those that use embedded proofs. An external proof is one that wraps an expression of this data model, such as via a JSON Web Token, which is elaborated on in the Securing Verifiable Credentials using JSON Web Tokens [[VC-JWT](https://w3c.github.io/vc-data-model/#bib-vc-jwt)] specification.

An `vc+ld+jwt` is a W3C Verifiable Credential, if you are asserting otherwise, I suggest we update the core spec first, since the securing spec is dependent on it for defining what "securing with an external proof" means...

I also doubt we will get consensus to say "a JWT cannot be a verifiable credential", either directly or indirectly.

Reply:
@TallTed @dlongley I'm resolving this thread based on https://github.com/w3c/vc-jwt/pull/123/commits/12eea9860400184442888e7eb978944b8d8b1b31 but feel free to suggestion additional changes in a new thread.